pylint/checkers/refactoring.py

```diff
@@ -331,12 +338,40 @@ def _check_superfluous_else_return(self, node):
         if _if_statement_is_always_returning(node) and not self._is_actual_elif(node):
             self.add_message('no-else-return', node=node)
 
+    def _check_consider_get(self, node):
+        def type_and_name_are_equal(node_a, node_b):
+            for _type in [astroid.Name, astroid.AssignName]:
+                if all(isinstance(_node, _type) for _node in [node_a, node_b]):
+                    return node_a.name == node_b.name
+            if all(isinstance(_node, astroid.Const) for _node in [node_a, node_b]):
+                return node_a.value == node_b.value
+            return False
+
+        if_block_ok = \
+            (isinstance(node.test, astroid.Compare)
+             and len(node.body) == 1
```

Comment:
You can also check that the operation is an `in` here before checking the rest so that it stops early if not.

Reply:
You mean an additional check like testing if "in" is the operation in node.test.ops ?

Reply:
Yup.